modules/manage/pages/manage-security/handle-certificate-errors.adoc

```diff
@@ -38,28 +46,24 @@ Recreate the certificate if necessary
 [#node-certificate-errors]
 == Node Certificate Errors
 
-The following error messages may be encountered when configuring the node
+The following error messages may be encountered when configuring and deploying the node
 certificate:
 
 |===
 | Couchbase Error Message | Description | Suggested User Action
 
-| `Cluster CA needs to be set before setting node certificate`
-| The cluster root certificate has not been established.
-| Set up the cluster CA certificate; then continue by creating the node certificate.
+| `Unable to read private key file <pathname>/inbox/pkey.key. The file does not exist.``
+| The private key is not in the `inbox` directory.
+| Ensure that the private key for the node certificate has been copied to the `inbox` folder of the current node.
+
+| `Unable to read private key file <pathname>/inbox/pkey.key. Missing permission for reading the file, or for searching one of the parent directories.``
+| The private key is in the `inbox` directory, but is not owned by user `couchbase`.
```

Comment:
I would say something like the following: Make sure the file is readable by user `couchbase`

Reply:
I have changed to:
| The private key is in the `inbox` directory, but is not readable by user `couchbase`.
| Ensure that the key is readable by user `couchbase`.